ep/lh/ep-lh-0020.md

```diff
@@ -0,0 +1,104 @@
+# linterhub: engine metadata
+
+| Metadata     |                                         |
+| ------------ |-----------------------------------------|
+| EP           | ep-lh-0020                              |
+| Version      | 1.0.0                                   |
+| Title        | linterhub: configuration file           |
+| Authors      | svoboda-rabstvo, binore                 |
+| Status       | Draft                                   |
+
+## Abstract
+
+This document describes the structure of configuration file used in lhconf as mentioned in [`ep-lh-0015`](#related).
+
+## Motivation
+
+Each developer have a lot of projects, and everyone can have special settings. This enhancement will unify how user can save special settings for each project.
+
+## Proposal
+
+Each project must have configuration file, called `.linterhub.json`. This file in JSON format which should describe:
+
+- `engines`, `ingores` -  It is a required set of attributes. It allows to understand how to run project with user settings.
+
+### JSON
+
+The configuration file must have the following properties:
+
+| Prorepty      | Type     | Required | Description |
+| -             | :-:      | :-:      | -           |
+| engines       | object[] | +        | How to run each engine in project |
+| ignores       | object[] | +        | Rules of ignore for run the project |
```

Comment:
I think it kinda of automatic translation, the meaning is lost. The project probably will not be executed (run) at all.